Lib/test/test_xml_etree_c.py

```diff
@@ -133,6 +134,22 @@ def test_setstate_leaks(self):
         self.assertEqual(len(elem), 1)
         self.assertEqual(elem[0].tag, 'child')
 
+    def test_iterparse_leaks(self):
+        # Test reference leaks in TreeBuilder (issue #35502).
```

Comment:
Maybe explain that the test is written to be executed using python3 -m test -R to ensure that the code doesn't leak?